pyccel/parser/semantic.py

```diff
@@ -58,6 +58,7 @@
 from pyccel.ast.core import Decorator
 from pyccel.ast.core import PyccelFunctionDef
 from pyccel.ast.core import Assert
+from pyccel.ast.core import Declare
```

Comment:
Declarations are added automatically for any variables which have been added to the scope. See https://github.com/pyccel/pyccel/blob/master/developer_docs/scope.md